docs/docusaurus/docs/cloud/about_gx.md

```diff
@@ -30,47 +32,38 @@ The following diagram provides an overview of the key GX Cloud architecture comp
 
 - **GX Cloud web UI** - A web interface that allows you to manage and validate your organization's data quality without running Python code and enables shared visibility into your organization's Validation Results and Checkpoint run history. It's browser- and platform-independent.
 
-- **GX Cloud API** - Provides a REST API to programmatically access and manage GX Cloud data and configurations. Both the GX open source software (OSS) client and the GX Agent use the GX Cloud API to query data from and send data to GX Cloud. Documentation for the GX Cloud API is not currently available, and it is not yet intended for use outside the GX OSS client and the GX Agent.
-
-- **GX Cloud message broker** - Enables communication between GX Cloud and the GX Agent.
-
-### Deployment components
-
-- **GX OSS client** - The Python library that powers GX Cloud and provides a Python client for programmatic access to GX Cloud. GX OSS contains the logic needed to test and document your organization's data, and you can also use it to create, manage, and interact with GX Cloud components.
-
-- **GX Agent** - A utility that runs in your organization's deployment environment. While running, the GX Agent can receive tasks generated from the GX Cloud web UI, such as running a Checkpoint or fetching Metrics, and execute these tasks against your Data Assets.
-
-## GX Agent
```

Comment:
Some of the headings you deleted, like this one, were used as anchors for links in the left ToC.  You should double check the ToC links for the pages you've edited and remove the ones that are no longer valid from `sidebars.js`.  I would suggest also adding a redirect to the `_redirects` file for those that have a suitable alternative (for instance, this one should redirect to your new "deploy_gx_agent.md" page.  You should be able to set up a redirect by following the pattern of other redirects in the file: on one line, first put the original URL (in this case the URL that is being removed from `sidebars.js`) and then, separated by a space, put the URL that should be displayed instead.

Reply:
Yup, the GX Agent sidebar heading should be removed, and the GX Cloud deployment patterns should be renamed & fixed to link to GX Cloud deployment options.